Makefile.Common

```diff
@@ -155,7 +155,7 @@ fmt: $(GOIMPORTS)
 
 .PHONY: lint
 lint: $(LINT) checklicense misspell
-	$(LINT) run --allow-parallel-runners --build-tags integration
+	$(LINT) run --allow-parallel-runners --build-tags integration --path-prefix $(shell basename "$(CURDIR)")
```

Comment:
Can we set `--path-prefix` on `.golangci.yml` instead in some way?

Reply:
I think setting `--path-prefix` on `.golangci.yml` may not be feasible, because the path prefix must be set dynamically for each component, the setting path-prefix in `.golangci.yml` can only be static. Unless we change the content of  `.golangci.yml` in each time we run the `golangci-lint run`